test.yml

```
@@ -0,0 +1,14 @@
+- name: Dummy Playbook with Hardcoded Password
+  hosts: all
+  become: yes
+  tasks:
+    - name: Create a dummy user with hardcoded password (bad practice)
+      user:
+        name: dummyuser
+        password: "MyHardCodedPassword123!"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)